src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -83,12 +83,13 @@ public bool CreateServiceMetadata(ServiceMetadata serviceMetadata)
         {
             string metadataAsJson = JsonConvert.SerializeObject(serviceMetadata);
             string orgPath = null;
+            string developerUserName = AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext);
 
             // TODO: Figure out how appsettings.json parses values and merges with environment variables and use these here.
             // Since ":" is not valid in environment variables names in kubernetes, we can't use current docker-compose environment variables
             orgPath = (Environment.GetEnvironmentVariable("ServiceRepositorySettings__RepositoryLocation") != null)
-                            ? Environment.GetEnvironmentVariable("ServiceRepositorySettings__RepositoryLocation") + serviceMetadata.Org.AsFileName()
-                            : _settings.RepositoryLocation + serviceMetadata.Org.AsFileName();
+                ? Environment.GetEnvironmentVariable("ServiceRepositorySettings__RepositoryLocation") + serviceMetadata.Org.AsFileName()
+                : _settings.GetOrgPath(serviceMetadata.Org.AsFileName(), developerUserName);
```

Comment:
developerUserName should probably also have AsFileName() if we are using this for org.